packages/itwinui-react/src/core/utils/components/Resizer.tsx

```diff
@@ -143,19 +143,23 @@ export const Resizer = (props: ResizerProps) => {
         }
         case 'right': {
           width = elementRef.current.style.width = `${initialWidth - diffX}px`;
+          height = elementRef.current.style.height = `${initialHeight}px`;
+          elementRef.current.style.transform = `translate(${translateX}px, ${translateY}px)`;
```

Comment:
need to update unit tests for these changes?

Reply:
weird that it was passing before then, but i'm guessing it has something to do with the dialog css present before resizing